### PR TITLE
Added documentation for AFINN, returned objects and tokenization.

### PR DESCRIPTION
**Pull request by therishabhpandey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,7 +79,62 @@ Yelp:    0.69 (+2%)
 ```
 
 ---
+### How it works
+#### AFINN 
+AFINN is a list of words rated for valence with an integer between minus five (negative) and plus five (positive). Sentiment analysis is performed by cross-checking the string tokens( words, emojis) with the AFINN list and getting their respective scores. The comparative score is simply: sum of each token / number of tokens. So for example let's take the following:
```
Comment (thisandagain) [inline README.md]: Couple minor nitpicks here:
- Fix spacing / formatting in parenthetical "tokens (e.g. words and emojis)"
- Format "sum of each token / number of tokens" as code by wrapping in `code`

File: README.md [hunk 2]
```diff
@@ -79,7 +79,62 @@ Yelp:    0.69 (+2%)
...
+In this case, love has a value of 3, allergic has a value of -2, and the remaining tokens are neutral with a value of 0. Because the string has 9 tokens the resulting comparative score looks like:
+(3 + -2) / 9 = 0.111111111
+
+This approach leaves you with a mid-point of 0 and the upper and lower bounds are constrained to positive and negative 5 respectively (the same as each token! 😸). For example, let's imagine an incredibly "positive" string with 200 tokens and where each token has an AFINN score of 5. Our resulting comparative score would look like this:
+
+(max positive score * number of tokens) / number of tokens
+(5 * 200) / 200 = 5
+
+#### Tokenization
+Tokenization works by splitting the lines of input string, then removing the special characters and finally splitting it using spaces. This is used to get list of words in the string.
```
Comment (thisandagain) [inline README.md]: Another really minor nitpick: 
- Please use an oxford comma for "characters, and finally"

File: README.md [hunk 3]
```diff
@@ -79,7 +79,62 @@ Yelp:    0.69 (+2%)
 ```
 
 ---
+### How it works
```
Comment (thisandagain) [inline README.md]: I think this whole section should probably be before "Benchmarks".  Also, please match the spacing between the H3 and the divider line as shown in the other sections:

```markdown
... lorem ipsum dolor sit amet.

---

### Some title
Lorem ipsum dolor sit amet...
```

File: README.md [hunk 4]
```diff
@@ -79,7 +79,62 @@ Yelp:    0.69 (+2%)
...
+    * __Words__: List of words from input string that were found in AFINN list. 
+    * __Positive__: List of postive words in input string that were found in AFINN list.
+    * __Negative__: List of negative words in input string that were found in AFINN list.
+    
+In this case, love has a value of 3, allergic has a value of -2, and the remaining tokens are neutral with a value of 0. Because the string has 9 tokens the resulting comparative score looks like:
+(3 + -2) / 9 = 0.111111111
+
+This approach leaves you with a mid-point of 0 and the upper and lower bounds are constrained to positive and negative 5 respectively (the same as each token! 😸). For example, let's imagine an incredibly "positive" string with 200 tokens and where each token has an AFINN score of 5. Our resulting comparative score would look like this:
+
+(max positive score * number of tokens) / number of tokens
```
Comment (thisandagain) [inline README.md]: Please wrap this and the line below it in a `code block`

File: README.md [hunk 5]
```diff
@@ -79,7 +79,62 @@ Yelp:    0.69 (+2%)
...
+* Returned Objects
+    * __Score__: Score calculated by adding the sentiment values of recongnized words.
+    * __Comparative__: Comparative score of the input string.
+    * __Token__: All the tokens like words or emojis found in the input string.
+    * __Words__: List of words from input string that were found in AFINN list. 
+    * __Positive__: List of postive words in input string that were found in AFINN list.
+    * __Negative__: List of negative words in input string that were found in AFINN list.
+    
+In this case, love has a value of 3, allergic has a value of -2, and the remaining tokens are neutral with a value of 0. Because the string has 9 tokens the resulting comparative score looks like:
+(3 + -2) / 9 = 0.111111111
```
Comment (thisandagain) [inline README.md]: Please wrap this line in a `code block`

**Comment by therishabhpandey** [comment]
I fixed these in #125. Please take a look.

**Pull request closed without merging**